microphone/drv_audio_pdm.c

```diff
@@ -101,8 +104,19 @@ static void drv_audio_pdm_event_handler(nrfx_pdm_evt_t const * const p_evt)
 			{
 				pdm_buf[l].released = true;
 				data_source_info.audio_source_info.audio_buffer = &pdm_buf[l].mic_buf;
+				if (l == 0 && timestamp_buffer[l] == 0) {
+					// First buffer, set timestamp to recording start timestamp
+					timestamp_buffer[l] = recording_start_timestamp;
+				} else {
+					// Subsequent buffers
+					// t_rec_i = t_release_i - (PDM_BUF_SIZE / effective_sampling_rate) * 1000
+					const uint32_t base_sample_rate = 20000; // 20kHz
```

Comment:
Can you still add a comment that I asked before? To add in this line https://github.com/TUDelft-SPC-Lab/midge-code/blob/master/microphone/drv_audio_pdm.c#L169 in this file that says that if the mode pdm_cfg.clock_freq is changed there, it should also be manually changed in this line too.